packages/flakybot/flakybot.go

```diff
@@ -70,7 +70,24 @@ func main() {
 	log.Println("Sending logs to Flaky Bot...")
 	log.Println("See https://github.com/googleapis/repo-automation-bots/tree/main/packages/flakybot.")
 
-	if ok := publish(cfg); !ok {
+	logs, err := findLogs(cfg.logsDir)
+	if err != nil {
+		log.Printf("Error searching for logs: %v", err)
+		os.Exit(1)
+	}
+	if len(logs) == 0 {
+		log.Printf("No sponge_log.xml files found in %s. Did you forget to generate sponge_log.xml?", cfg.logsDir)
+		os.Exit(0)
```

Comment:
I debated and am open to changing it. It might not be an error if there is no `sponge_log.xml` file? But, on the other hand, why would you call `flakybot` if you didn't have logs?

Reply:
I think we want to have the binary throw a failing return exit code otherwise maintainers won't notice

Reply:
Done.